.github/workflows/boulder-ci.yml

```diff
@@ -73,6 +73,10 @@ jobs:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
+      
+      # TODO(#6998): Remove this step.
+      - name: Update docker compose plugin
+        run: mkdir -p ${DOCKER_CONFIG:-$HOME/.docker}/cli-plugins && curl -SL https://github.com/docker/compose/releases/download/v2.20.0/docker-compose-linux-x86_64 -o ${DOCKER_CONFIG:-$HOME/.docker}/cli-plugins/docker-compose && chmod +x ${DOCKER_CONFIG:-$HOME/.docker}/cli-plugins/docker-compose
```

Comment:
Can you link to some documentation on what the cli-plugins directory does? Also this looks like the command is copy-pasted from somewhere, which is fine, but we should link to it.

Reply:
They're from https://docs.docker.com/compose/install/linux/#install-the-plugin-manually. Comment updated. The cli-plugins directory is just a place where the docker command looks for executables that it can expose as subcommands, much like `git` searches $PATH for executables named `git-something` and exposes them as `git something` subcommands.